runtime/src/accounts_db.rs

```diff
@@ -3032,20 +3031,29 @@ impl AccountsDb {
     }
 
     pub fn shrink_candidate_slots(&self) -> usize {
-        let _guard = self.active_stats.activate(ActiveStatItem::Shrink);
-
         let shrink_candidates_slots =
```

Comment:
Can we move the local  var `shrink_candidates_slots` down to the scope where it is used?